daily_schoolar/client/src/App.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import "./App.css";
-import Admin from "./Components/AdminPage/Admin";
+import Admin from "./Components/AdminPage/Index";
```

Comment:
Hey Jon, from my understanding, you can drop the ' /Index ' from the import line, but if it's working we should be good. I'll confirm that as well :-)